app/models/incoming_text_message.rb

```diff
@@ -21,11 +21,11 @@
 class IncomingTextMessage < ApplicationRecord
   include ContactRecord
 
-  belongs_to :client, touch: true
+  belongs_to :client
   validates_presence_of :body
   validates_presence_of :received_at
 
-  before_create { client.touch(:last_response_at) }
+  before_create { client.touch(:updated_at, :last_response_at) }
```

Comment:
I personally found it confusing to explicitly update `updated_at` here, and would prefer just `client.touch(:last_response_at)`, but this is non-blocking and if you like it more this way, then I can 100% live with it this way!